examples/chem-sync-local-flask/README.md

```diff
@@ -54,25 +54,28 @@ curl localhost:8000/health
 
 If Flask is running, you should see `OK` printed.
 
-Be sure to note the URL created for you by `localtunnel`. The log line should look something like this:
+Be sure to note the URL created for you by `Cloudflare Tunnel`. The log line should look something like this:
 
 ```
-app-workshop-local-tunnel-1   | your url is: https://brave-wombats-poke.loca.lt
+cloudflare-tunnel-1  | +--------------------------------------------------------------------------------------------+
+cloudflare-tunnel-1  | |  Your quick Tunnel has been created! Visit it at (it may take some time to be reachable):  |
+cloudflare-tunnel-1  | |  https://brave-wombats-poke.trycloudflare.com                                              |
+cloudflare-tunnel-1  | +--------------------------------------------------------------------------------------------+
 ```
 
 On *nix systems, you can easily obtain _just_ the URL via:
 
 ```
-docker compose logs local-tunnel | grep -o https://.* | tail -n 1
+docker compose logs cloudflare-tunnel | grep -o https://.* | tail -n 1
```

Comment:
I'm comparing to @damola-benchling 's original commit to switch over to `cloudflare-tunnel` on the workshop branch https://github.com/benchling/app-examples-python/commit/f86fb9511ae3b0196acbdce450748961f9234357#diff-49b6ba22abe964e9ebdf2bba9044fa1381e9099171a62aa75c3c1def9222dd8aL173

this line might need to be 

```suggestion
docker compose logs cloudflare-tunnel | grep -o 'https://[^ ]*trycloudflare.com[^ ]*' | tail -n 1
```